indicator_templates/quarto/3_societal_impact/introduction_societal_impact.qmd

```diff
@@ -0,0 +1,73 @@
+---
+author:
+    - name: T. Venturini
+      orcid: 0000-0003-0004-5308
+      affiliations:
+      - ref: cis-cnrs
+      - ref: ugen
+
+affiliations:
+- id: cis-cnrs
+  name: Centre national de la recherche scientifique
+  department: Centre for Internet and Society
+  city: Paris
+  country: France
+- id: ugen
+  name: University of Geneva
+  city: Geneva
+  country: Switzerland
+---
+
+# Uptake in and impact on to societal issues {#uptake-in-and-impact-on-to-societal-issues .unnumbered}
+
+<div>
+
+## History
+
+| Version | Revision date | Revision    | Author       |
+|---------|---------------|-------------|--------------|
+| 1.0     | 2024-11-14    | First draft | T. Venturini |
+
+</div>
+
+## Open Science Beyond Academia: An Exploration of Societal Uptake and Impact
+
+Open Science is increasingly recognized as essential for advancing academic research and industrial innovation. Understanding the full impact of Open Science, however, requires looking beyond the boundaries of academia and Industry examining its utilization in various societal sectors. This section of the handbook considers how diverse stakeholders engage with Open Science, the challenges in measuring its influence, and the diverse approaches being taken to assess its real-world effects.
+
+Societal impacts are arguably the trickiest impact to assess. Decades of research in the social sciences have shown that direct causal explanations are at best simplistic and at worst misleading. At least three excellent reasons exist to be wary of the very idea of *societal impacts*:
```

Comment:
This is put a bit too strongly, I think. Yes, there are clear limitations, and evidence is being over interpreted causally, but in some cases, social scientists also do have clear causal evidence, and this now completes wipes that off the table.

In addition, I think that being "wary" of the idea of societal impacts is also perhaps too strong. Perhaps "critical" would be more appropriate?